src/main/scala/redis/api/Clusters.scala

```diff
@@ -70,4 +70,26 @@ case class ClusterSlots() extends RedisCommand[MultiBulk,Seq[ClusterSlot]] {
     }
 
   }
+}
+
+case class ClusterInfo() extends RedisCommand[Bulk, scala.collection.immutable.Map[String, String]] {
+  val isMasterOnly = false
+  val encodedRequest: ByteString = encode("CLUSTER INFO")
+  def decodeReply(b: Bulk): scala.collection.immutable.Map[String, String] = {
+    b.response.map(_.utf8String.split("\r\n").map(_.split(":")).map(s => (s(0),s(1))).toMap).getOrElse(Map.empty)
+  }
+  override val decodeRedisReply: PartialFunction[ByteString, DecodeResult[Bulk]] = {
+    case s => RedisProtocolReply.decodeReplyBulk(s)
+  }
+}
+
+case class ClusterNodes() extends RedisCommand[Bulk, Array[scala.collection.immutable.Map[String, String]]] {
+  val isMasterOnly = false
+  val encodedRequest: ByteString = encode("CLUSTER NODES")
+  def decodeReply(b: Bulk): Array[scala.collection.immutable.Map[String, String]] = {
+    b.response.map(_.utf8String.split("\n").map(_.split(" ")).map(s => scala.collection.immutable.Map("id" -> s(0), "ip-port" -> s(1), "flags" -> s(2), "master" -> s(3), "ping-sent" -> s(4), "pong-recv" -> s(5), "config-epoch" -> s(6), "link-state" -> s(7), "slots" -> s.drop(8).mkString(" ")))).getOrElse(Array.empty)
```

Comment:
I could, but it is arguable that which way is better.

The redis.io document regarding the INFO (https://redis.io/commands/info) has the following notes:

Please note depending on the version of Redis some of the fields have been added or removed. A robust client application should therefore parse the result of this command by skipping unknown properties, and gracefully handle missing fields.

Using a case class might introduce version incompatibility issues within the client API.

If you insist, I will make the changes. But until then, I will hold off the changes for now.